tools/appliance/systemvmtemplate/scripts/configure_systemvm_services.sh

```diff
@@ -108,6 +108,19 @@ function configure_services() {
   systemctl disable strongswan
   systemctl disable x11-common
   systemctl disable xl2tpd
+  systemctl disable vgauth
+  systemctl disable sshd
```

Comment:
This may cause error in somecases, ideally we want sshd enabled in all cases.

Reply:
My recollection is that sshd is actually only started after the firewall rules are established, otherwise there will be a period of vulnerability during startup.